services/QuillLMS/spec/services/vitally_integration/serialize_vitally_sales_user_spec.rb

```diff
@@ -261,6 +264,49 @@
     )
   end
 
+  it 'presents evidence data' do
+    new_student = create(:user, role: 'student')
+    create(:classrooms_teacher, user: teacher, classroom: classroom)
+    create(:classrooms_teacher, user: teacher, classroom: old_classroom)
+    create(:students_classrooms, student: student, classroom: classroom)
+    create(:students_classrooms, student: old_student, classroom: old_classroom)
+    create(:students_classrooms, student: new_student, classroom: classroom)
+    classroom_unit.assigned_student_ids << new_student.id
+    classroom_unit.save!
+
+    evidence_unit_activity = create(:unit_activity, :evidence_unit_activity, unit: unit)
+    create(:activity_session,
+      classroom_unit: classroom_unit,
+      activity: evidence_unit_activity.activity,
+      user: student,
+      state: 'finished',
+      completed_at: Time.now - 10.days
+    )
+    create(:activity_session,
+      classroom_unit: classroom_unit,
+      activity: evidence_unit_activity.activity,
+      user: student,
+      state: 'finished',
+      completed_at: Time.now - 3.days
+    )
+    create(:activity_session,
+      classroom_unit: classroom_unit,
+      activity: evidence_unit_activity.activity,
+      user: new_student,
+      state: 'started',
+      created_at: Time.now - 1.year,
+      completed_at: Time.now - 1.year
+    )
+
+    teacher_data = SerializeVitallySalesUser.new(teacher).data
+
+    expect(teacher_data[:traits]).to include(
+      evidence_activities_assigned_this_year: 2,
+      evidence_activities_completed_this_year: 2,
+      date_of_last_completed_evidence_activity: (Time.now - 3.days).strftime("%F")
```

Comment:
might be a good idea to freeze the `Time.now` into a local variable, to avoid a possible race condition with the other invocations of `Time.now`